scripts/fixate-requirements.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import os
+import os.path
+import sys
+from distutils.version import StrictVersion
+
+
+cwd = os.path.abspath(os.getcwd())
+FIXED_REQF = os.path.join(cwd, '../fixed-requirements.txt')
```

Comment:
Adding a module level docstring which describes what this script does and how it works would be good.

We should probably also use more verbose variable names and clarify what the variable contains in a comment. For example:

```python
 # Requirements file which contains a list of dependencies which are common to all the components.
# Those global requirements file is then merged with package local requirements files to form a final
# requirements file for the component (package).
GLOBAL_REQUIREMENTS_FILE = ...
```